src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,124 @@
+
+[Code Fix: The SIBA Website's Move from Create-React-App to Astro](https://brittanisavery.com/post/move-siba-to-astro) - Read about converting a legacy CRA project to Astro by Brittani S. Avery
+
+[Translate routes in Astro with Content collection, subpage, optional trailingSlash and base support](https://www.webdesign-sopelnik.de/en/blog/translate-routes-for-astro-content-collections-or-subpages-with-trailingslash-and-base-support/) - A guide to translating routes for content collections and subpages by Artur Sopelnik 
+
+## Showcase
+
+Each week on the Astro Community Call in Discord, we shine the spotlight on projects posted by community members in our `#showcase` channel from the following week. Here's a sample of what we saw this month!
+
+### Tools
+
+- [Darkmatter](https://getdarkmatter.dev/) - A CMS-like interface for your Astro content collections (MacOS)
+- [Frontmatter CMS](https://frontmatter.codes/) -  A Headless CMS right in your code editor - now supports Astro!
+- [Search Astro Documentation Raycast Extension](https://www.raycast.com/ElianCodes/search-astro-docs) - search Astro docs directly within Raycast
+- [Set Up a Static Site With Astro on Kinsta](https://kinsta.com/docs/astro-static-site/) - new guide for setting up a static site with Kinsta’s Application or Static Site hosting.
+
+### Themes
+
+- https://github.com/AminoffZ/catonaut by @aminoff
+- https://github.com/kevinzunigacuellar/astro-supabase by @kevinzunigacuellar
+- https://astro.build/themes/details/engineering-blog/ by @eddyvinck
+
+### Websites
+
+- https://www.noobscience.rocks/ by @noobscience
+- https://fadhelmurphy.github.io/ by @mamangpro
+- https://rohitk06.vercel.app/ by @rohit06
+- https://www.hvlxh.tk/ by @hitesh_dev
+- https://astronvim.com/ by @uzaaft
+- https://www.minov.studio/en/ by @suhaylmv
+- https://www.linwood.dev/ by @codedoctor
+- https://shamann.dev/ by @sylvhama
+- https://paualbert.netlify.app/ by @akond
+- https://procastinatordev.vercel.app/ by @procastinatordev
+- https://www.mcdle.net/ by @elitogame
+- https://www.decimal.fm/ by @chrsgrrtt
+- https://dev.jeromeabel.net/ by @jeromeabel
+- https://www.nunogois.com/ by @nunogois
+- https://ovidius-astro-theme.netlify.app/ by @tomas_ba
+
+### Starlight in the wild
+
+Need inspiration for your documentation site? Here are the new Starlight sites we spotted this month!
+
+- https://docs.corsace.io/en/
+- https://certs.msfthub.wiki/
+- https://aoi.js.org/
+- https://www.chproxy.org/
+- https://www.bunshi.org
+- https://drops-of-php.hi-folks.dev/
+- https://angular-challenges.vercel.app
+- https://astro-docs-docs.netlify.app/
+- https://docs.cataclysmbn.org/en/
+- https://www.reatom.dev/
+- https://zsh-test-runner-docs.vercel.app/
+- https://ngxtension.netlify.app/getting-started/introduction/
+- https://skyscript.js.org/
+- https://nostalgist.js.org/
+- https://reactnativeunistyles.vercel.app/
+- https://sibiraj-s.github.io/ngx-editor
+- https://webmonetization.org/docs/
```

Comment:
Including the full URLs here is pretty hard to read and also not great for screenreaders. Can we add proper link text with the title of the site/theme or at least shorter URLs so e.g. `https://ngxtension.netlify.app/getting-started/introduction/` becomes `[ngxtension.netlify.app](https://ngxtension.netlify.app/getting-started/introduction/)`? (Site title would be ideal IMO)